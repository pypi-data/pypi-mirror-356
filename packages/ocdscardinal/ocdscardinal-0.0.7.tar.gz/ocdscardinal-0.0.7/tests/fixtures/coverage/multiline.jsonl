{"a":{"b":0,"c":[{"d":[0,1]}]}}
{"a":{"b":0}}
{"x":0}
{"x":0}
