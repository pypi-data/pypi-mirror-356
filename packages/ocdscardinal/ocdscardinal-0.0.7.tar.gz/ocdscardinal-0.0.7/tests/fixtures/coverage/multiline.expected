{"":4,"/":4,"/a":2,"/a/":2,"/a/b":2,"/a/c":1,"/a/c[]":1,"/a/c[]/":1,"/a/c[]/d":1,"/a/c[]/d[]":2,"/x":2}
